tools/js-test-runner/jest-config/jest.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+let config = {};
+
+const defaultConfig = {
+	preset: '@wordpress/jest-preset-default',
+	rootDir: process.cwd(),
+	testEnvironment: 'jsdom',
+	// TODO: remove as soon gutenberg release new version
+	//https://github.com/WordPress/gutenberg/blob/%40wordpress/jest-preset-default%407.1.5-next.33ec3857e2.0/packages/jest-preset-default/jest-preset.js
+	testMatch: [ '**/__tests__/**/*.[jt]s?(x)', '**/test/*.[jt]s?(x)', '**/?(*.)test.[jt]s?(x)' ],
+	moduleNameMapper: {
+		'\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': path.join(
+			__dirname,
+			'./fileMock.js'
+		),
+		'\\.module\\.(css|less)$': 'identity-obj-proxy',
```

Comment:
What's going on here? We don't seem to have any files named like this.

If we can drop this, we could drop the added dep on `identity-obj-proxy` too.

Reply:
@anomiex We already have this kind of file `.module.css`, this is related to our new approach going into CSS Modules. This is a example: https://github.com/Automattic/jetpack/blob/master/projects/js-packages/components/components/admin-page/style.module.scss ... I just forgot to update for `.css` or `.scss`.